test/integration/connect/envoy/Dockerfile-bats

```diff
@@ -1,6 +1,6 @@
 FROM docker.mirror.hashicorp.services/fortio/fortio AS fortio
 
-FROM docker.mirror.hashicorp.services/bats/bats:1.6.0
```

Comment:
Why did you need to bump this?

Reply:
See above https://github.com/hashicorp/consul/pull/13159#issuecomment-1133395868